HerPublicWebsite.BusinessLogic/Models/LocalAuthorityData.cs

```diff
@@ -4,37 +4,38 @@ namespace HerPublicWebsite.BusinessLogic.Models;
 
 public class LocalAuthorityData
 {
-        public enum Hug2Status
-        {
-                NotTakingPart,
-                Pending,
-                Live
-        }
+    public enum Hug2Status
```

Comment:
I think the branch history is a bit tricky already, given the way these various tickets are spawning off each other.

I agree that this would be better in general, but it might be overly messy in this case.